packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexPageHeader.tsx

```diff
@@ -47,6 +68,12 @@ export const RecordIndexPageHeader = () => {
   return (
     <PageHeader title={pageHeaderTitle} Icon={Icon}>
       <PageHotkeysEffect onAddButtonClick={handleAddButtonClick} />
+      <PageFavoriteFoldersDropdown
+        record={view}
+        dropdownId={FAVORITE_FOLDERS_DROPDOWN_ID}
+        objectNameSingular="view"
+        isFavorite={isFavorite}
+      />
```

Comment:
You should add the dropdown in the ActionMenu also (Cmd+K)
See `packages/twenty-front/src/modules/action-menu/actions/record-actions/components/ManageFavoritesActionEffect.tsx` L48

![image](https://github.com/user-attachments/assets/a08ee73c-6a8d-429b-ae76-21814b4d40e5)

Reply:
seen with @Bonapara we don't want any dropdown here